README.adoc

```diff
@@ -308,14 +307,14 @@ include::finish/resources/CustomConfigSource.json[]
 ----
 
 InventoryManager.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright tags=add_fallback']
```

Comment:
- don't think `tags=add_fallback` is necessary
- also extra blank line appearing on top of file in codepane

Reply:
addressed.